api/net/Middleware/ErrorHandlingMiddleware.cs

```diff
@@ -145,11 +146,12 @@ private async Task HandleExceptionAsync(HttpContext context, Exception ex)
             }
             else if (ex is AuthenticationException)
```

Comment:
AuthenticationException was referencing invalid exception.  Should be the one from Microsoft.  Deleted the custom one.